lib/server.go

```diff
@@ -92,10 +98,14 @@ func (s *HTTPServer) handleCmd(res http.ResponseWriter, req *http.Request) {
 }
 
 func (s *HTTPServer) handleEvents(res http.ResponseWriter, req *http.Request) {
+  enableCors(&res)
   fmt.Fprintln(os.Stderr, "/events")
 
   r := eventlogReader(req.Context())
-  io.Copy(res, r)
+  scanner := bufio.NewScanner(r)
+  for scanner.Scan() {
+    fmt.Fprintf(res, "data: %s\n\n", scanner.Text())
+  }
```

Comment:
After this for loop, we should be sure to check if the `scanner` [errored](https://golang.org/pkg/bufio/#Scanner.Err):

```go
if err := scanner.Err(); err != nil {
  fmt.Fprintln(os.Stderr, "reading standard input:", err)
}
```